js/bybit.js

```diff
@@ -665,6 +825,24 @@ module.exports = class bybit extends Exchange {
         return this.milliseconds () - this.options['timeDifference'];
     }
 
+    async isUnifiedMarginEnabled (params = {}) {
+        const enableUnifiedMargin = this.safeValue (this.options, 'enableUnifiedMargin');
+        if (enableUnifiedMargin === undefined) {
+            const response = await this.privateGetUserV3PrivateQueryApi (params);
+            const result = this.safeValue (response, 'result', {});
+            const permissions = this.safeValue (result, 'permissions', {});
+            this.options['enableUnifiedMargin'] = this.safeInteger (permissions, 'unified') === 0;
+        }
+        return this.options['enableUnifiedMargin'];
+    }
+
+    async upgradeUnifiedAccount (params = {}) {
+        // warning this method can only be called once
+        // it is not reverseable and you will be stuck with a unified margin account
+        // you also need at least 5000 USDT in your bybit account to do this
+        return await this.privatePostUnifiedV3PrivateAccountUpgradeUnifiedAccount (params);
```

Comment:
@sc0Vu  user won't see this warning by default, so I'd suggest doing something similar to `createMarketBuyRequiresPric` here, i.e, throw an exception by default warning the user and that exception can be disabled in the options

Reply:
@carlosmiei do you want to check account balance of user?

Reply:
@sc0Vu nop, just a simple warning stating this action is irreversible (you can add the 5k info too) so that users won't enable it accidentally